usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/port/ExecParamRepository.java

```diff
@@ -35,15 +35,8 @@
     String URL_KEY = "url";
     String INPUT_KEY = "input";
     String EXCLUSION_KEY = "exclude";
-    String PATHWAY_MIN_LENGTH_KEY = "pathway_min_length";
```

Comment:
why were those removed?

Reply:
They were not used actually, I'll bring them back when implementing semantic validation of field `pathways.length`.